systemtest/src/main/java/io/strimzi/systemtest/templates/crd/KafkaTemplates.java

```diff
@@ -246,6 +259,28 @@ public static KafkaBuilder kafkaFromYaml(File yamlFile, String clusterName, int
             .endSpec();
     }
 
+    private static KafkaBuilder addImagePullSecret(KafkaBuilder builder) {
+        List<LocalObjectReference> imagePullSecrets = Collections.singletonList(new LocalObjectReference(Environment.SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET));
+        PodTemplate podTemplate = new PodTemplate();
+        podTemplate.setImagePullSecrets(imagePullSecrets);
+        KafkaClusterTemplate kafkaClusterTemplate = new KafkaClusterTemplateBuilder().withPod(podTemplate).build();
+        ZookeeperClusterTemplate zookeeperClusterTemplate = new ZookeeperClusterTemplateBuilder().withPod(podTemplate).build();
+        EntityOperatorTemplate entityOperatorTemplate = new EntityOperatorTemplateBuilder().withPod(podTemplate).build();
+
+        builder.editSpec()
+            .editKafka()
+                .withTemplate(kafkaClusterTemplate)
+            .endKafka()
+            .editZookeeper()
+                .withTemplate(zookeeperClusterTemplate)
+            .endZookeeper()
+            .editEntityOperator()
+                .withTemplate(entityOperatorTemplate)
+            .endEntityOperator()
+            .endSpec();
```

Comment:
```suggestion
        builder
            .editSpec()
                .editKafka()
                    .withTemplate(kafkaClusterTemplate)
                .endKafka()
                .editZookeeper()
                    .withTemplate(zookeeperClusterTemplate)
                .endZookeeper()
                .editEntityOperator()
                    .withTemplate(entityOperatorTemplate)
                .endEntityOperator()
            .endSpec();
```
Just indent again

Reply:
For kafka, I switched to use the env variable based image-pull-secret injection and removed the template editing code.